doc/classes/EditorInspector.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="EditorInspector" inherits="ScrollContainer" version="4.0">
 	<brief_description>
+		A tab used to edit the properties of the selected.
 	</brief_description>
 	<description>
+		The editor inspector is by default located on the right-hand side of the editor. It is used to edit the properties of the node selected. For example, you can select a node such as the sprite2D then edit it's transform through the inspector tool. The editor inspector is an essential tool in the game development workflow.
```

Comment:
"A tab used to edit the properties of the selected" incomplete sentence, add the word "node".

"node selected" -> "selected node", "sprite2D" -> "Sprite2D" and remove "the", "it's -> its"

Reply:
Thanks for pointing this out, I will fix it.

Reply:
Fixed the changes, can you please review it one more time?